### ISSUE TITLE
quantum bridge limited channels?

### ISSUE THREAD

**Issue by ggblade** [AppliedEnergistics/Applied-Energistics-2#1390]
my computer is already a good 4+ years old, i cant handle having everything in my base. so to preserver cpu on my computer and using more cpu on the server i am using chunk loaders and have automatic systems away from my base. this saves my fps. done some farming areas, mod spawning areas... so i  thought i should start with a ore processing + autocrafting area. build the rooms and made a quantum bridge. however for this i need alot of channels. seeying as im doing this purely with AE, i ended with around 70 channels. but trying to connect 3 dense cable to the quantum bridge does not work, only one is being used, one is using 32/32 but the others dont use anything. so i switched it up a bit. using ME tunnels, though if i can only bring a max of 32 channels, using me tunnels should give me the amount i need. however its not giving the channel to the ME tunnel. now its giving 4 channels, 3 to my crafting units, and 1 to a crafting terminal. meaning the cable has 28 more channels and theres only 4 ME tunnels it should power. anyone have a idea on why this is?

in short. the quantum bridge is not giving a channel to a P2P Tunnel - me. is this normal? is there a way to bypass it? or any way to have 70+ channels going trough a quantum bridge seeying as the limit seems to be 32

**Comment by XFactHD** [comment]
You need to put one p2p tunnel on each side of the quantum bridge. Like that:
|\____QNB____/|
|/                        \|
P2P                   P2P

**Comment by ggblade** [comment]
http://imgur.com/LJbuOt2,6EfHpGu,qI1D7Ql
i have them on each side of the quantum bridge.
on the first picture is my quantum bridge, red dense brings channels for the black me tunnel. 

second image is my controllers, its the same red dense cable wich controlls all me tunnels. showing the 4 me tunnels that are being used at the other side of the quantum bridge, clearly having all channels available, and on the left bot you can see the black cable thats being used to bring channels into the quantum bridge.

third picture shows black cable going to the different me tunnels. its only giving channels to crafting units and crafting terminal.

**Comment by XFactHD** [comment]
I think the p2p on the first picture is in the wrong direction, just turn it around and replace the red smart cable with a red dense cable. This should work.

**Comment by ggblade** [comment]
afaik dense cables cant connect to me tunnels, well atleast not to the back of it. also the first picture is the connection from the "main" system. turning that around would mean no connections would go in.
well just to be sure il try it

**Comment by ggblade** [comment]
well ive tried and as thought, doesnt work

**Comment by ggblade** [comment]
to try and show a bit more of what im talking about took some extra pictures.

http://imgur.com/KZSQaPx,6e8JMMh,3CbEkLR,eTTVpGD,1w78ZGF,b1WMGYl,S8gqtdt,DkJRIY9,36gP4O9

picture 1:
my controllers already showed them before, the top black one powers the me tunnels, then the 4 colors next to it are then ones im trying to use on my crafting/processing.

Picture 2:
the black cable that connects to the bridge from the main base side. clearly only using 4 channels.

Picture 3:
same black cable as in picture 2, still on tha side of the main base. connecting it to the quantum bidge.

picture 4:
on the other side of the quantum bridge, showing that the me tunnel doesnt receive channels.

picture 5: 
this tunnel is next to the previous picture. also not receiving channels.

picture 6:
showing that the cable is receiving 3 out of the 4 channels, this cable is connected to the 2 me tunnels in the previous picture.

picture 7:
the 3 crafting units that are receiving a "channel".

picture 8:
the crafting terminal, also receiving a channel.(4/4)

picture9:
another 2 me tunnels that arent receiving channels.

**Comment by thatsIch** [comment]
generally you want to put the P2P tunnels to wrap around the QNB, so you get 32x32 channels through them.

* A QNB can hold 32 channels
* A P2P Tunnel can tunne 32 channels by applying a dense cable
* If you put 32 P2P tunnels onto the QNB, you can tunnel 32 dense cables through 1 QNB
